BigriverBookstore-apple.xcodeproj/xcuserdata/viorelCornel.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -9,6 +9,26 @@
 			<key>orderHint</key>
 			<integer>6</integer>
 		</dict>
+		<key>BigriverBookstore-appleTests.xcscheme_^#shared#^_</key>
```

Comment:
@virgildicu We still have files which are specific to your local Xcode. Please check the gitignore is matching it.